.appveyor.yml

```
@@ -1,4 +1,4 @@
-version: 4.3.0.99.{build}
+version: 4.4.0
```

```suggestion
version: 4.4.0.{build}
```

oops (this is necessary to keep getting new builds)